test/WovnIndexSampleTest.php

```diff
@@ -46,9 +46,12 @@ public function testDetectMultipleFiles () {
     $this->assertEquals('This is index.html', $this->runWovnIndex('/'));
   }
 
-  public function testInvalidPath () {
+  /**
+   * @runInSeparateProcess
+   */
+  public function testInvalidPathBlip () {
     $this->touch('index.php');
-    $this->assertEquals('This is index.php', $this->runWovnIndex('/../../index.php'));
+    $this->assertEquals('Page Not Found', $this->runWovnIndex('/../../index.php'));
```

Comment:
/../../index.php should be an invalid directory when appended to root, right?

Also, this fails unless @runInSeparateProcess is appended???